### DISCUSSION TITLE
参考文献原本的序号排序不显示

### DISCUSSION THREAD

**Discussion by lqwhhhlqw** [sjtug/SJTUThesis#886] [提问]
@article{moisy,
author = {Moisy, Frédéric and Rabaud, Marc and Salsac, Kévin},
year = {2009},
month = {06},
pages = {1021-1036},
title = {A synthetic Schlieren method for the measurement of the topography of a liquid interface},
volume = {46},
journal = {Experiments in Fluids},
doi = {10.1007/s00348-008-0608-z}
}
比如这个在正文中使用\cite{moisy}，但是未显示序号，只会在右上角显示小字号的moisy

**Comment by lqwhhhlqw** [comment]
请问下是因为什么原因呢，是因为部分文件确实或是？

**Comment by LogCreative** [comment]
说明参考文献没有正确编译，检查是否正确使用了 latexmk 编译。

**Reply by LogCreative** [reply]
可以提供日志信息以供参考。

**Reply by lqwhhhlqw** [reply]
\addbibresource{bibdata/thesis.bib} 显示未找到这个文件？

**Reply by LogCreative** [reply]
这个文件是存在于模板中的，如果你使用了其他数据库，需要改成其他名称。

**Reply by lqwhhhlqw** [reply]
\addbibresource{bibdata/thesis.bib}
\addbibresource{bibdata/the.bib}
这样也是的，是因为未找到这个文件，所以引用不了？

**Reply by lqwhhhlqw** [reply]
就是 一直显示 未找到 bib这个文件

**Reply by LogCreative** [reply]
请将报错信息完整地展示出来，至少以截图的方式。

**Comment by lqwhhhlqw** [comment]
\addbibresource{bibdata/thesis.bib} 显示未找到这个文件？
